.github/workflows/ci.yml

```diff
@@ -1,10 +1,6 @@
 name: Testing CI
 
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+on: [pull_request]
```

Comment:
I'm not an expert of Github workflows but I thought merging PR to main branch is "push" event. So given this config, we will not have a builds after merging and deployment would not happen.

Reply:
you're right